tensorizer/serialization.py

```diff
@@ -3266,6 +3333,7 @@ def __init__(
         *,
         encryption: Optional[EncryptionParams] = None,
         limit_cpu_concurrency: Optional[int] = None,
+        max_tensors: Optional[int] = None,
```

Comment:
Can there be added docs on this?

Reply:
Yeah agreed there should be some docs on this

Reply:
Right now refer to my info in the PR description itself, but we should include something like that in the code